Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -68,7 +68,11 @@ struct NoMagicNumbersRule: SwiftSyntaxRule, OptInRule, ConfigurationProviderRule
                 let a = Int(3)
             }
             class MyTest: XCTestCase {}
-            """)
+            """),
+            Example("let foo = 1 << 2"),
+            Example("let foo = 1 >> 2"),
+            Example("let foo = 2 >> 2"),
+            Example("let foo = 2 << 2")
```

Comment:
Negative numbers like in `-2 << 2` and `2 >> -2` would still trigger. Is this intentional out of being a bit questionable?

Reply:
So that was not intentional, but I think I could live with it, on the basis that negative numbers are quite a rare case.

What do you think?

Reply:
Negative number used in shift operations can be considered "magic" I think. 😉

Reply:
Agreed... a negative number in a shift operation seems like it would be an error case, actually...  So, if we warn on it, we're just making the world a better place! 😄